templates/ldap_config.sql.j2

```diff
@@ -0,0 +1,57 @@
+-- MySQL dump 10.15  Distrib 10.0.31-MariaDB, for debian-linux-gnu (x86_64)
```

Comment:
Is there no other way to configure this?
All values are set in two different ways here.
I mean, there has to be an option to insert or replace, like `UPSERT` in postgresql.

Reply:
there is not